arbitrary-users-patch/base_images

```diff
@@ -10,5 +10,5 @@ che-nodejs8-centos      registry.centos.org/che-stacks/centos-nodejs
 che-php-7               quay.io/eclipse/che-php-base:7.4
 che-python-3.6          centos/python-36-centos7:1
 che-python-3.7          python:3.7.4-slim
-che-quarkus             quay.io/quarkus/centos-quarkus-maven:19.2.1
+che-quarkus             quay.io/quarkus/centos-quarkus-maven:20.0.0-java8
```

Comment:
maybe we could bump to Java11 version as well ? (before it was only providing java8)

Reply:
Do you want to use Java11 instead of 8 ? or provide both alternatives in our base image ?